src/index.js

```diff
@@ -65,7 +65,7 @@ async function format(options) {
     // Let prettier infer the parser using the filepath, if present. Otherwise
     // assume the file is JS and default to the babel parser.
     filePath ? { filepath: filePath } : { parser: 'babel' },
-    getPrettierConfig(filePath, prettierPath),
+    await getPrettierConfig(filePath, prettierPath) || {},
```

Comment:
Why you refactor to move `|| {}` here?

Reply:
So we don't have to await in `getPrettierConfig`. As I look at it, a ternary in `getPrettierConfig` might be even better.

```
return prettier.resolveConfig
  ? prettier.resolveConfig(filePath)
  : {};
```

What do you think?

Reply:
That makes more sense to me.

Reply:
```suggestion
    await getPrettierConfig(filePath, prettierPath),
```